ocs_ci/deployment/ibm.py

```diff
@@ -26,206 +30,174 @@ def destroy_ocs(self):
-        # provisioners. other than noobaa.
-
-        # STEP 1 & 4
-        # Get a list of storage classes
-        storage_classes = ocp.OCP(kind="StorageClass").get()
-        scnoprovisioner = []
-        scopenshiftprovisioner = []
-        for storage_class in storage_classes.get("items", []):
-            name = storage_class["metadata"]["name"]
-            if storage_class["provisioner"] == "kubernetes.io/no-provisioner":
-                scnoprovisioner.append(name)
-            else:
-                scopenshiftprovisioner.append(name)
-
-        storage_clusters = ocp.OCP(
-            kind="StorageCluster", namespace=cluster_namespace
-        ).get()
-        sclassindevicesets = []
-        for storage_cluster in storage_clusters.get("items", []):
-            device_sets = storage_cluster["spec"]["storageDeviceSets"]
-            sclassindevicesets.append(
-                [i["dataPVCTemplate"]["spec"]["storageClassName"] for i in device_sets]
-            )
 
-        # sclassindevicesets is a list of lists. Each outer list is for
-        # a specific storage cluster. All storage classes in the inner
-        # list are same (same storage class used across all devicesets
-        # in a storage cluster. So pick only the first one.
-
-        # for each storage class, get LocalVolume (owner).
-
-        if len(sclassindevicesets) == 0:
-            logger.warning("No storage class in device sets. Quitting.")
-            return
-
-        # compare scnoprovisioner and sclassindevicesets
-        tmplist = []
-        for sclist in sclassindevicesets:
-            for sc in sclist:
-                if sc in scnoprovisioner:
-                    if sc not in tmplist:
-                        tmplist.append(sc)
-
-        scandlvlist = []
-        for sc in tmplist:
-            storage_class = ocp.OCP(kind="StorageClass", resource_name=sc)
-            localvolumemeta = storage_class.get().get("metadata")
-            localvolume = localvolumemeta["labels"][
-                "local.storage.openshift.io/owner-name"
-            ]
-            scandlvlist.append((sc[0], localvolume))
-
-        alldevicepaths = []
-        for lv in scandlvlist:
-            # Get localvolume and corresponding devices
-            localvolume = (
-                ocp.OCP(
-                    kind="LocalVolume",
-                    namespace=local_storage_namespace,
-                    resource_name=lv[1],
-                )
-                .get()
-                .get("spec")
+        # https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/deploying_openshift_container_storage_using_bare_metal_infrastructure/assembly_uninstalling-openshift-container-storage_rhocs
+
+        # Section 3.1 Step 1
+        # Deleting PVCs
+        rbd_pvcs = [
+            p
+            for p in pvc.get_all_pvcs_in_storageclass("ocs-storagecluster-ceph-rbd")
```

Comment:
we can use constants here to get storageclassname https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L95